sound/soc/codecs/hdac_hdmi.c

```diff
@@ -1915,6 +1929,7 @@ static void hdmi_codec_complete(struct device *dev)
 	struct hdac_device *hdev = dev_to_hdac_dev(dev);
 	struct hdac_hdmi_priv *hdmi = hdev_to_hdmi_priv(hdev);
 
+	snd_hdac_display_power(hdev->bus, hdev->addr, true);
```

Comment:
@libinyang is this needed if you are already powering off the display power after codec_prepare?